epigraphhub/data/get_data.py

```diff
@@ -0,0 +1,253 @@
+import pandas as pd
+from sqlalchemy import create_engine
+
+engine_public = create_engine(
+    "postgresql://epigraph:epigraph@localhost:5432/epigraphhub"
+)
+engine_private = create_engine(
+    "postgresql://epigraph:epigraph@localhost:5432/privatehub"
```

Comment:
it would be better if it gets this URI from an environment variable instead of a fixed one inside the code